examples/inspection/plot_permutation_importance_multicollinear.py

```diff
@@ -8,26 +8,30 @@
 The :class:`~sklearn.ensemble.RandomForestClassifier` can easily get about 97%
 accuracy on a test dataset. Because this dataset contains multicollinear
 features, the permutation importance will show that none of the features are
-important. One approach to handling multicollinearity is by performing
-hierarchical clustering on the features' Spearman rank-order correlations,
-picking a threshold, and keeping a single feature from each cluster.
+important. Hierarchical clustering is used to visualize the correlations
```

Comment:
CorrelationThreshold does not implement hierarchical clustering, does it? Rather it is a screening procedure.

FeatureGrouping can be used for hierachical cluserting, I believe.

Reply:
Agreed. I am thinking of leaving this example alone and keep it using hierarchical clustering.

Reply:
Well, I think that the reason to used CorrelationThreshold is for interpretation purposes, hence it makes sense to have it in the example.

What I would recommend is to have the examples with three parts:
- Raw permutation importances
- FeatureGrouping permutation importances
- CorrelationThreshold permutation importances

and a discussion of the caveats of selection and interpretation.